extensions/notebook-renderers/src/index.ts

```diff
@@ -205,6 +205,7 @@ function initializeScroll(scrollableElement: HTMLElement, disposables: Disposabl
 		scrollableElement.scrollTop = scrollTop !== undefined ? scrollTop : scrollableElement.scrollHeight;
 		scrollableElement.addEventListener('scroll', onScrollHandler);
 		disposables.push({ dispose: () => scrollableElement.removeEventListener('scroll', onScrollHandler) });
+		scrollableElement.tabIndex = 0;
```

Comment:
if we don't add a tabIndex here, it will just use the outer container, within the `focusFirstFocusableOrContainerInOutput` function. That way we don't have to do this for every output type.